docs/source/services/timelock_service/index.rst

```diff
@@ -0,0 +1,38 @@
+.. _external-timelock-service:
+
+External Timelock Service
+=========================
+
+.. warning::
+
+   Running external timestamp and lock services is currently an experimental feature.
+   If you are interested in testing it, please contact the AtlasDB team.
+
+.. danger::
+
+   Improperly configuring one's cluster to use external timestamp and lock services can result in **SEVERE DATA
+   CORRUPTION**! Please contact the AtlasDB team if you wish to try this feature.
```

Comment:
Okay, I've merged the warnings. Once this is the recommended it won't be there anymore, as you say.